code/modules/mob/living/carbon/combat.dm

```diff
@@ -27,9 +27,10 @@
 		return FALSE
 	target_zone = null
 	var/power = I.force
+	if(isgrey(user))
```

Comment:
do this OOP

check a var on the species

Reply:
I don't know how

Reply:
no you should add the power modifier to the abstract type /datum/species in species.dm

Reply:
I have no idea how to do that